scripts/db.sh

```diff
@@ -12,7 +12,8 @@ fi
 COMMAND=$1
 shift
 if [ "$COMMAND" != "dump" ] && [ "$COMMAND" != "import" ] && [ "$COMMAND" != "drop_and_import" ]; then
-  echo "Unrecognized command: $COMMAND. Must be one of dump, import, drop_and_import."
+  echo "Unrecognized command: $COMMAND."
+  print_usage_and_exit
   exit 1
```

Comment:
Since we just called `print_usage_and_exit`, there's no need for this `exit 1` anymore.